src/js/msp/MSPHelper.js

```diff
@@ -435,6 +435,12 @@ MspHelper.prototype.process_data = function(dataHandler) {
                 GPS_RESCUE.throttleHover     = data.readU16();
                 GPS_RESCUE.sanityChecks      = data.readU8();
                 GPS_RESCUE.minSats           = data.readU8();
+                if (semver.gte(CONFIG.apiVersion, "1.43.0")) {
+                    GPS_RESCUE.ascendRate            = data.readU16();
+                    GPS_RESCUE.descendRate           = data.readU16();
+                    GPS_RESCUE.allowArmingWithoutFix = data.readU8() !== 0;
```

Comment:
I think it is better to keep this the way it was sent, since we are pushing it back in https://github.com/betaflight/betaflight-configurator/pull/1812/files#diff-1c62b9c8a0f16c83d475e2b2c301dd3dR1718 even if it was not modified - otherwise this will break in a hard to diagnose way if it is ever changed from a boolean to something else.

Reply:
Do you say get rid of the boolean value and use and int? We have both versions in others MSP commands, but if it is preferred to use an int, I will change it.
I added some of them too to my other approved PR https://github.com/betaflight/betaflight-configurator/pull/1810 so I think I will need to change it there... 😋

Reply:
Yes it is better - in some instances we are 'feeding through' data when sending an MSP command to save in a tab that does not modify all of the data covered in the command, and that has led to unintended consequences in the past when the data format was changed, and configurator did not pass it through exactly as it was received.